### PR TITLE
Add travis docker build stages

### PR DESCRIPTION
**Pull request by philbrookes**
Add travis docker build stages

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,15 +64,7 @@ generate:
...
 compile:
-	go build -o=gitea-operator ./cmd/manager/main.go
-
-.PHONY: packr
-packr:
-	packr
-
-.PHONY: packr-clean
-packr-clean:
-	packr clean
```
Comment (mikenairn) [inline Makefile]: Did you intend to remove these packr targets?

File: Makefile [hunk 2]
```diff
@@ -28,12 +28,17 @@ setup:
 	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 	@echo Installing errcheck
 	@go get github.com/kisielk/errcheck
-	@echo Installing packr
-	@go get -u github.com/gobuffalo/packr/packr
```
Comment (mikenairn) [inline Makefile]: Is this not being used?

File: Makefile (hunk 1 above)
Comment (mikenairn) [reply Makefile]: nvm, i see this now https://github.com/integr8ly/gitea-operator/pull/22. I guess this was copied from elsewhere and had these left in it.

**Comment by philbrookes** [comment]
@pb82 let me know if you have any thoughts on this.

**Pull request merged**